src/v/config/configuration.cc

```diff
@@ -1652,6 +1652,20 @@ configuration::configuration()
       "Number of attempts metadata operations may be retried.",
       {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
       5)
+  , cloud_storage_attempt_cluster_recovery_on_bootstrap(
+      *this,
+      "cloud_storage_attempt_cluster_recovery_on_bootstrap",
+      "If set to `true`, when a cluster is started for the first time and "
```

Comment:
How does this work once the cluster is recovered? Should user set it back to false or it just becomes a noop?

Reply:
This will only do anything on the first startup of the cluster, when it's bootstrapped. After this, this property doesn't do anything (until the next time a new cluster is brought up using the same bucket).